horovod/common/global_state.h

```diff
@@ -110,6 +110,9 @@ struct HorovodGlobalState {
   // benefit from a smaller chunk size.
   int64_t adasum_mpi_chunk_size = 1<<30;
 
+  // Enable use of batched d2d memcopy kernel on GPU
+  bool batch_d2d_memcopies = false;
```

Comment:
Is the plan to change the default to true after more testing is done to verify performance improvements?  Is there a chance that performance could degrade with this setting?

Reply:
I think we should enable this by default, though it might be good to allow it to be disabled via the env var just in case people see performance degradations. I haven't seen any performance degradations in my testing on V100 cards. I will update the PR to default this to true. This will also make sure testing is run using the new kernel, since right now it is not picking it up.